ui/k-toolbar/main.kytos

```diff
@@ -0,0 +1,276 @@
+<template>
+  <k-toolbar-item icon="search-location" tooltip="Napp sdntrace_cp">
+    <div class="scroll">
+      <k-accordion>
+        <k-accordion-item title="Trace">
+
+          <k-accordion-item title="Required Switch Parameters">
+
+            <k-input-auto id="dpid" :value.sync="dpid"
```

Comment:
I'll recommend that similarly to `in_port` you also use a `k-input-auto` and `k-input` pair, just so we have the autocompletion for the `dpid` that we currently have, but then you keep track of the value on the `k-input`. 

The autocompletion of the dpid, should be `metadata - dpid` or `data_path - dpid` as a fallback if metadata isn't defined or empty, and then you always keep track of the `dpid` to send later on in the payload.